lib/src/material_controls.dart

```diff
@@ -379,7 +373,6 @@ class _MaterialControlsState extends State<MaterialControls>
 
     setState(() {
       if (controller.value.isPlaying) {
-        playPauseIconAnimationController.reverse();
```

Comment:
Fun fact: While `CupertinoControls` had an animated icon, it never called these methods on its controller, therefore never showing an animatoin.

Reply:
Haha, great that it's working now :D